optional mqtt authentication

mherwege opened the pull request:
Signed-off-by: Mark Herwege <mark.herwege@telenet.be>

jac459 committed:
1.0.1

jac459 committed:
Delete BroadlinkAC.json

jac459 committed:
Delete BroadlinkTV.json

jac459 committed:
Delete KODI.json

jac459 committed:
Delete LGTV.json

jac459 committed:
Delete Volumio.json

jac459 committed:
Delete naim.json

jac459 committed:
Delete philipsHue.json

jac459 committed:
1.0.1

jac459 committed:
Delete philipsHueGroup.json

jac459 committed:
Delete roon.json

jac459 committed:
Delete yamaha.json

jac459 committed:
Add files via upload

jac459 committed:
Add files via upload

jac459 committed:
Add files via upload

jac459 committed:
Add files via upload

jac459 committed:
Add files via upload

jac459 committed:
Add files via upload

jac459 committed:
1.0.1

jac459 committed:
1.1.0

jac459 committed:
1.1.0

jac459 committed:
Add files via upload

jac459 committed:
1.1.0

jac459 committed:
Add files via upload

jac459 committed:
v1.0.3

jac459 committed:
1.0.4

xml2js missing fix.

jac459 committed:
Add files via upload

jac459 committed:
Meta v1.0.5

label helper Matt support + processing manager with XML Post

jac459 committed:
minor bugs correction

jac459 committed:
Added git installation

jac459 committed:
Update installMeta.sh

MarkusMas committed:
Added "Running the meta in pm2" + cleaned up a bit

MarkusMas committed:
Update update.sh

MarkusMas committed:
Update drivers.manifest

MarkusMas committed:
Update README.md

MarkusMas commented:
Sounds like a valuable addition :)
I think we should merge it. But i better leave that up to to jac as he needs to decide were to add thise changes (branch: master and/or release)

MarkusMas committed:
Update latest dependency versions

MarkusMas committed:
removed downloading package-lock.json

MarkusMas committed:
Labels ar now exposed over mqtt

same as change in master branch 16.01.2022

jac459 committed:
shell scripts to launch meta from pm2 without logs

MarkusMas committed:
Updated Driver Verison Numbers (matched to master)

Updated Driver Verison Numbers:
- enigma2 to Version 8
- naim to Version 3
- Shelly 1 / 1PM / 1L to Version 2
- Shelly 2.5 (Roller Shutter mode) to Version 2
- Yamaha MusicCast to Version 17

MarkusMas committed:
Updated Driver Verison Numbers (matched to master)

Updated Driver Verison Numbers:
- enigma2 to Version 8
- naim to Version 3
- Shelly 1 / 1PM / 1L to Version 2
- Shelly 2.5 (Roller Shutter mode) to Version 2
- Yamaha MusicCast to Version 17

jac459 committed:
Minor bugs correction

Added missing MQTT wiring for labels and images.
Logs correction

jac459 committed:
1.1

New Vault, minor bug fixes, speed improvement, androidTV support.

jac459 committed:
Update README.md

jac459 committed:
Update README.md

jac459 committed:
Update README.md

jac459 committed:
Update update.sh

Include libraries

jac459 committed:
Merge pull request #5 from jac459/jac459-patch-1

Update update.sh

jac459 committed:
Update update.sh

jac459 committed:
Update installMeta.sh

Upgrade node.js to V20

jac459 committed:
Update package.json

Correction of missing binary search

jac459 committed:
Delete drivers.manifest

jac459 committed:
v1.1.1

Fix to accept unauthorized connection (required for WebOS)
